src/WebPageTranscoder.js

```diff
@@ -21,10 +21,12 @@ import {ErrorReporter} from './util/ErrorReporter.js';
 export const scriptSelector = 'script[type="module"],script[type="text/traceur"]';
 
 export class WebPageTranscoder {
-  constructor(url) {
+  constructor(url = document.location.href,
+      traceurOptions = $traceurRuntime.options) {
```

Comment:
I used the `$traceurRuntime.options` because this change will break anyone relying on globals. But ok, I removed them.